app/views/shared/_skip_links.html.slim

```diff
@@ -1,5 +1,5 @@
 - unless cookies_preference_set?
   a.govuk-skip-link href="#cookie-consent" Skip to cookie consent
 a.govuk-skip-link href="#main-content" Skip to main content
```

Comment:
Is it worth having this as an `else` for the `if content_for?(:skip_links)` instead of always showing it?